pkg/manager/manager.go

```diff
@@ -484,7 +487,13 @@ func (s *FraktiManager) ListImages(ctx context.Context, req *kubeapi.ListImagesR
 	errs := []error{}
 
 	imageServiceList := []runtime.ImageService{s.hyperImageService, s.privilegedImageService}
-	imageMapList := make([]map[string]*kubeapi.Image, 2)
+	workerNum := 2
+	// NOTE: make unikernel image service the last one
```

Comment:
Does the order care?

Reply:
This order only influnce the order of which runtime image stores in `imageMapList`, we relay on this order in following steps, seems this comment a little bit distracting.